app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ContinuousScanActivity.java

```diff
@@ -322,35 +337,35 @@ private void showOfflineSavedDetails(OfflineSavedProduct offlineSavedProduct) {
         } else {
             name.setText(R.string.productNameNull);
         }
-        if (productDetails.get("add_brands") == null || productDetails.get("add_brands").equals("")) {
-            brand.setText(R.string.productBrandNull);
-        } else {
-            brand.setText(productDetails.get("add_brands"));
-        }
-        if (productDetails.get("quantity") == null || productDetails.get("quantity").equals("")) {
-            quantity.setText(R.string.productQuantityNull);
-        } else {
-            quantity.setText(productDetails.get("quantity"));
-        }
-        if (productDetails.get("image_front") != null) {
-            Picasso.with(ContinuousScanActivity.this)
-                    .load("file://" + productDetails.get("image_front"))
-                    .error(R.drawable.placeholder_thumb)
-                    .into(productImage, new Callback() {
-                        @Override
-                        public void onSuccess() {
-                            imageProgress.setVisibility(View.GONE);
-                        }
-
-                        @Override
-                        public void onError() {
-                            imageProgress.setVisibility(View.GONE);
-                        }
-                    });
-        } else {
-            productImage.setImageResource(R.drawable.placeholder_thumb);
-            imageProgress.setVisibility(View.GONE);
-        }
+//        if (productDetails.get("add_brands") == null || productDetails.get("add_brands").equals("")) {
```

Comment:
In the future if we are at anytime trying to uncomment the code then it will be a mess with single line comments for each line . In java you can use `/* .........*/`  for multiline comments. It is a good practice to use multiline comments for cases such as these.